tests/integration/source-code-build/enable-ts-loader.test.ts

```diff
@@ -22,7 +22,7 @@ describe('source build', () => {
     original: string;
   };
 
-  beforeAll(async () => {
+  beforeEach(async () => {
```

Comment:
this issue can also be resolved by reuse the Page instance and it will be faster:

![89f61b18-422c-4464-8aff-1b1cba343ba7](https://github.com/web-infra-dev/modern.js/assets/7237365/381d28f0-6e13-4806-b6bc-27b1af27947e)